tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -57,24 +57,30 @@ function HeaderComp() {
               <div className="usa-nav__secondary">
                 <ul className="usa-nav__secondary-links">
                   <li className="usa-nav__secondary-item">
-                    <a href="/">
-                      <FontAwesomeIcon
-                        className="margin-right-1"
-                        icon={faUserCircle}
-                      />
-                      {user.email}
-                    </a>
+                    {user && user.email && (
```

Comment:
In order for tabbing to work in navigation, I couldn't have the entirety of the `usa-nav__secondary` component be dynamically rendered. Instead, I changed the `Sign Out` button to change between `Sign In` and `Sign Out` depending on if the user is authenticated and only show the profile secondary link on authentication. This fixes a11y for keyboard navigation on the menu.